baseplate/metrics.py

```diff
@@ -313,7 +313,7 @@ def replace(self, new_value):
 def make_client(namespace, endpoint):
     """Return a configured client.
 
-    :param str namespace: The root key to namespace all metrics under.
+    :param str namespace: The root key to prefix all metrics with.
```

Comment:
The prefix is how we implement a namespace, given the hierarchical nature of graphite metrics. Calling it a prefix is a practical explanation of effect from the abstract name. I'm not sure I see what the deeper meaning of namespace is here.